hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,82 +10,95 @@ This module conforms to the [Comportable guideline for peripheral functionality.
 
 At a high-level, this ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the entropy source into a FIFO that can be read from the TLUL bus, or sent out through a hardware interface.
-The rng source will generate a relative slow set of bit streams, in the tens of kilohertz range.
+The RNG source will generate a relatively slow set of bit streams, in the tens of kilohertz range.
 The digital source can be programmed to be faster, or much slower if desired.
-A digital source rate of flexiable speed might be useful in demonstrating how the downstream applications use the entropy.
-
-Once the threshold has been reached for collected entropy bits, the firmware has two options of notification: an interrupt, or polling.
-The `es_entropy_valid` bit will be set when the {{< regref "ES_THRESH" >}} register value is equal or less than the number of elements in the TLUL FIFO.
-Optionally, an interrupt may be a better use of CPU cycles.
+A digital source rate of flexible speed might be useful in demonstrating how the downstream applications use the entropy.
+
+Once a noise source has been selected through configuration, the raw noise data processing begins.
+This ENTROPY_SRC block will do health checks, collect and pack bits, and finally send them through a conditioning unit. 
+There are two modes in which entropy bits are delivered, bypass and FIPS/CC mode. 
+Bypass mode will deliver faster bits for chip boot usage, and the bits are not required to be FIPS/CC compliant entropy.
+This means that the min-entropy value is a relatively lower and acceptable value for bypass mode.
+There are several test statistics that are kept in registers to provide diagnostic capabilities when examining real entropy test fails.
 For the hardware interface, entropy bits will be transferred whenever the downstream consumer is ready to receive them.
 
 ## Compatibility
 This IP block does not have any direct hardware compatibility requirements.
-The firmware routines, as described by SP 800-90B, must be supported.
-However, as long as the function provided by the register set supports all of the call parameters, any compatibility requirements will be met.
+However, the general design of this block follows the overall NIST recommendation, as described by the SP 800-90B documentation.
 
 # Theory of Operations
 
 As already described, this IP block will collect bits of entropy for firmware or hardware consumption.
-This revision supports both an LFSR for the digital implementation, and an external interface for the rng implementation.
-For the LRSR implementation, any polynomial may be used so long as it is primitive in Galois Field of order 2, or GF(2).
-However, preference should be given to polynomials with the fewest number of terms.
+This revision supports both an LFSR for the digital implementation, and an external interface for the RNG implementation.
```

Comment:
This revision supports both an LFSR for the pseudo-random digital implementation, and an external interface for the physical noise source implementation.